release/release.go

```diff
@@ -1019,36 +1027,19 @@ func linuxRelease(v version.Version) {
 			continue
 		}
 
-		if task.Variant == "ubuntu-race" {
+		if pf.Variant() != task.Variant {
 			continue
 		}
 
-		_, knownVariant := platform.GetByVariant(task.Variant)
-		if !knownVariant {
-			log.Fatalf("found dist task with unknown variant '%s'\n", task.Variant)
-		}
-
 		distTasks = append(distTasks, task)
 	}
 
-	if len(distTasks) != platform.Count() {
-		log.Fatalf(
-			"found %d dist tasks, but expected %d release platforms",
-			len(distTasks), platform.Count(),
-		)
+	if len(distTasks) != 1 {
```

Comment:
It might not; this is me being defensive. I'm not sure how it would show up in the API if you were to restart a task, which is why I kept the assertion